common.mk

```diff
@@ -135,7 +136,7 @@ busybox-cleaner-common:
 LINUX_COMMON_FLAGS ?= LOCALVERSION= CROSS_COMPILE=$(CROSS_COMPILE_NS_KERNEL)
 
 linux-common: linux-defconfig
-	$(MAKE) -C $(LINUX_PATH) $(LINUX_COMMON_FLAGS)
+	$(MAKE) -C $(LINUX_PATH) -j$(NPROC) $(LINUX_COMMON_FLAGS)
```

Comment:
We can skip this commit for now, while trying to figure out how to solve this in a proper way. I had a gut feeling that there might be questions about it, so therefore I made sure to not combine that with anything else.